axiom/tests/test_txn.c

```diff
@@ -130,6 +130,22 @@ nrapp_t* nr_app_verify_id(nrapplist_t* applist NRUNUSED,
   return p->txns_app;
 }
 
+const char* nr_app_get_host_name(const nrapp_t* app) {
```

Comment:
Isnt this already defined in nr_app.c?

Reply:
@mfulb Please see 87a2a7bc533b489f784c68e8fd655f00f16f0abf commit message for an explanation why this is required to be defined here too.